Sources/AEPOptimize/Optimize.swift

```diff
@@ -139,6 +128,43 @@ public class Optimize: NSObject, Extension {
         return true
     }
 
+    /// Processes the propositions request event, dispatched with type `EventType.optimize` and source `EventSource.requestContent`.
+    ///
+    /// It processes events based on the type of the requested event type
```

Comment:
nit: Can we simplify this comment to something like this? `It processes events based on the `requesttype` of the event`